browser/ui/tabs/split_view_tab_strip_model_adapter.cc

```diff
@@ -110,17 +110,18 @@ void SplitViewTabStripModelAdapter::OnTabMoved(
   // together
   auto tab_handle =
       model_->GetTabHandleAt(model_->GetIndexOfWebContents(move->contents));
-  if (!split_view_browser_data_->IsTabTiled(tab_handle)) {
+
+  auto tile = split_view_browser_data_->GetTile(tab_handle);
+  if (!tile.has_value()) {
```

Comment:
Can you not check `if (!tile) {` ?

Reply:
Yeah, I seem to be more like "literal" person haha. Will change